content/ui-patterns/messaging.mdx

```diff
@@ -7,9 +7,9 @@ import {Box} from '@primer/react'
 
 Primer includes three different messaging components:
 
-- [Toasts](https://primer.style/css/components/toasts)
-- [Flash alerts](https://primer.style/css/components/alerts)
-- [Popovers](https://primer.style/css/components/popover)
+- Toasts
```

Comment:
I think we should either add a link or remove this until we add toasts to PRC or PVC